ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/AffectedImagesTable.tsx

```diff
@@ -0,0 +1,223 @@
+                    <Th sort={getSortParams('Fixable')}>
+                        Fix status
+                        {isFiltered && <DynamicColumnIcon />}
+                    </Th>
+                    <Th sort={getSortParams('Operating System')}>Operating system</Th>
+                    {/* TODO Add sorting for these columns once aggregate sorting is available in BE */}
+                    <Th>
+                        Affected components
+                        {isFiltered && <DynamicColumnIcon />}
+                    </Th>
+                    <Th>First discovered</Th>
+                </Tr>
+            </Thead>
+            {images.map(
+                (
+                    { id, name, operatingSystem, scanTime, topImageVulnerability, imageComponents },
+                    rowIndex
+                ) => {
+                    const topSeverity =
+                        topImageVulnerability?.severity ?? 'UNKNOWN_VULNERABILITY_SEVERITY';
+
+                    const isFixable = topImageVulnerability?.isFixable ?? false;
+                    const FixabilityIcon = isFixable ? FixableIcon : NotFixableIcon;
+
+                    const SeverityIcon = SeverityIcons[topSeverity];
+                    const severityLabel = vulnerabilitySeverityLabels[topSeverity];
+                    const isExpanded = expandedRowSet.has(id);
+
+                    return (
+                        <Tbody key={id} isExpanded={isExpanded}>
+                            <Tr>
+                                <Td
+                                    expand={{
+                                        rowIndex,
+                                        isExpanded,
+                                        onToggle: () => expandedRowSet.toggle(id),
+                                    }}
+                                />
+                                <Td dataLabel="Image">
+                                    {name ? (
+                                        <Flex
+                                            direction={{ default: 'column' }}
+                                            spaceItems={{ default: 'spaceItemsNone' }}
+                                        >
+                                            <Button
+                                                variant={ButtonVariant.link}
+                                                isInline
+                                                component={LinkShim}
+                                                href={getEntityPagePath('Image', id)}
+                                            >
+                                                {name.remote}
+                                            </Button>{' '}
+                                            <span className="pf-u-color-400 pf-u-font-size-sm">
+                                                in {name.registry}
+                                            </span>
```

Comment:
i'll probably be factoring this out into a separate component fwiw since i'll use it in the overview images table too